### ISSUE TITLE
No way to create single files.

### ISSUE THREAD

**Issue by mtancoigne** [el-cms/superBake#16]
For now there's no way to create single files (as elements, error pages, ...).

This would be great to have this in order to create layouts files and other page elements dependant of the theme.

**Comment by mtancoigne** [comment]
This should be done with a **_file**_ section in the configuration file :

``` YAML
file:
  path: path::to::snippet
  options:
     option1: value1
     option2: value2
     ...
```

**Comment by mtancoigne** [comment]
Implemented in 0.3 alpha2
